lib/manageiq/providers/workflows/engine.rb

```diff
@@ -43,10 +43,11 @@ def self.floe_docker_runner
             port = ENV.fetch("KUBERNETES_SERVICE_PORT")
 
             Floe::Workflow::Runner::Kubernetes.new(
-              "server"     => URI::HTTPS.build(:host => host, :port => port).to_s,
-              "token_file" => "/run/secrets/kubernetes.io/serviceaccount/token",
-              "ca_cert"    => "/run/secrets/kubernetes.io/serviceaccount/ca.crt",
-              "namespace"  => File.read("/run/secrets/kubernetes.io/serviceaccount/namespace")
+              "server"                   => URI::HTTPS.build(:host => host, :port => port).to_s,
+              "token_file"               => "/run/secrets/kubernetes.io/serviceaccount/token",
+              "ca_cert"                  => "/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+              "namespace"                => File.read("/run/secrets/kubernetes.io/serviceaccount/namespace"),
+              "workflow_service_account" => "manageiq-default"
```

Comment:
This value can be productized using the app name in the CR, so it may not always be "manageiq"

Reply:
Moved to an ENV var AUTOMATION_JOB_SERVICE_ACCOUNT